specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,547 @@
+          "type": "string"
+        },
+        "equals": {
+          "description": "The value of the event's field will be compared to this value (case-insensitive) to determine if the condition is met.",
+          "type": "string"
+        },
+        "containsAny": {
+          "description": "The value of the event's field will be compared to the values in this array (case-insensitive) to determine if the condition is met.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ActionList": {
+      "type": "object",
+      "description": "A list of Activity Log Alert rule actions.",
+      "properties": {
+        "actionGroups": {
+          "description": "The list of the Action Groups.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ActionGroup"
+          },
+          "x-ms-identifiers": [
+            "actionGroupId"
+          ]
+        }
+      }
+    },
+    "ActionGroup": {
+      "type": "object",
+      "description": "A pointer to an Azure Action Group.",
+      "properties": {
+        "actionGroupId": {
+          "description": "The resource ID of the Action Group. This cannot be null or empty.",
+          "type": "string"
+        },
+        "webhookProperties": {
+          "type": "object",
+          "description": "the dictionary of custom properties to include with the post operation. These data are appended to the webhook payload.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "actionProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Predefined list of properties and configuration items for the action group."
+        }
+      },
+      "required": [
+        "actionGroupId"
+      ]
+    },
+    "TenantAlertRulePatchObject": {
+      "description": "An Activity Log Alert rule object for the body of patch operations.",
+      "properties": {
+        "tags": {
+          "description": "The resource tags",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/TenantAlertRulePatchProperties",
+          "description": "The activity log alert settings for an update operation."
+        }
+      }
+    },
+    "TenantAlertRulePatchProperties": {
+      "description": "An Activity Log Alert rule properties for patch operations.",
+      "properties": {
+        "enabled": {
+          "description": "Indicates whether this Activity Log Alert rule is enabled. If an Activity Log Alert rule is not enabled, then none of its actions will be activated.",
+          "type": "boolean",
+          "default": true
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=5&length=13)

Recommend getting this via $ref of ErrorResponse in v5 common types.

Reply:
Align with ActivityLogAlerts

Reply:
Are the schemas different in any way that matters to your clients?